drivers/storage/portworx/manifest/manifest.go

```diff
@@ -240,6 +246,14 @@ func fillDefaults(
 	if rel.Components.NodeWiper == "" {
 		rel.Components.NodeWiper = defaultNodeWiperImage
 	}
+	if rel.Components.DynamicPlugin == "" {
```

Comment:
Can you add these after line 224 as well? That is used when we are not able to connect to the spec generator for some reason.

Reply:
added